dnf-plugins-core.spec

```diff
@@ -415,6 +415,29 @@ Download module metadata from all enabled repositories, module artifacts and pro
 repository.
 %endif
 
+%if %{with python3}
+%package -n python3-dnf-plugin-system-upgrade
+Summary:        System Upgrade Plugin for DNF
+Requires:       python3-systemd
+Provides:       dnf-command(system-upgrade)
+Provides:       dnf-command(offline-upgrade)
+Provides:       dnf-command(offline-distrosync)
+Provides:       dnf-plugin-system-upgrade = %{version}-%{release}
+Provides:       python3-dnf-plugins-extras-system-upgrade = %{version}-%{release}
```

Comment:
Shouldn't this be `python3-dnf-plugins-core-system-upgrade = %{version}-%{release}`?